app/views/backups/status.js.erb

```diff
@@ -0,0 +1,6 @@
+<% if @backup.running? %>
+    new ServerPoller("<%= status_server_app_backup_path(@backup.app.server, @backup.app, @backup) %>").poll();
+<% else %>
+  var backup = $("#<%= dom_id(@backup) %>");
```

Comment:
No need to assign this var imo. Also a bit inconsistent use of semicolons #nitpicking